packages/grant-explorer/src/features/api/rounds.ts

```diff
@@ -93,31 +96,29 @@ async function fetchRoundsByTimestamp(
   }
 }
 
-function getActiveChainIds() {
-  const activeChainIds: string[] = [];
-  const isProduction = process.env.REACT_APP_ENV === "production";
+/**
+ * Filters out testnet chainIds in production, otherwise returns all active chainIds
+ * */
+const getActiveChainIds = () =>
+  Object.keys(ChainId).filter((chainId) => {
```

Comment:
I see `allChains` is defined elsewhere as:

```ts
export const allChains: Chain[] =
  process.env.REACT_APP_ENV === "development"
    ? [...testnetChains(), ...mainnetChains()]
    : [...mainnetChains()];
```

So there seems to be a test/non-testnet selection logic already. Does this change need to take that into account?

Unrelated to this PR: would it make sense to try and derive `ChainId` from `allChains` instead of having the id's defined both as enum and as keys in `allChains`? (This reminds me of the Jimmy Koppel's color example about hidden coupling.)

Reply:
true - i didn't realize allChains already has this functionality - this function is therefore superfluous and we can just use the allChains array directly.

second point - it could, but I believe this wouldn't work at comptime, meaning that we would lose the static checking, and ChainId would become just a number, because we don't know the contents of the allChains object at comptime. but maybe `as const` could help here.

Reply:
Good point. It _might_ become a number union, which _might_ still be ok, but if it became just a `number`, that would be a step back.